Tools/python/Sample.py

```diff
@@ -17,6 +17,7 @@
     'T2_US_Florida': 'root://cmsio5.rc.ufl.edu/',
     'T2_US_MIT': 'root://xrootd.cmsaf.mit.edu/',
     'T2_US_UCSD': 'root://redirector.t2.ucsd.edu/',
+    'global_redirect': 'root://cms-xrd-global.cern.ch/',
```

Comment:
Shouldn't we be using the cmsxrootd.fnal.gov redirector before the global one? https://twiki.cern.ch/twiki/bin/view/CMSPublic/WorkBookXrootdService#ReDirector

Actually I assume it typically is more efficient to use CRAB before the global redirector, too.

Reply:
I used the global one because some experts on lpc-howto suggested not to use specific site directors but use the global one. Just for curiosity, why do we specify sites of samples for submitting jobs? For the global director, I think you are right, the cmsxrootd.fnal.gov should be the better one to use. 
Now I find that there are many times specifying a given site doesn't really work well because for some samples, no site have all the files, they are just spread over several sites, using global redirector or CRAB is really helpful in those cases. I found that happening a lot when I run over UL samples so I actually prefer CRAB now.

Reply:
> Just for curiosity, why do we specify sites of samples for submitting jobs?

Good question--it may not be needed anymore... I think it used to be the case that it was faster this way, but the cmsxrootd.fnal.gov redirector may be better nowadays, maybe with a sanity check before submitting things that the samples are somewhere that we can easily access from FNAL (otherwise we could end up trying to access samples that would be slow to use at FNAL).

If we can just set everything to use a single redirector instead of specifying specific sites (without any loss in speed of running things), it would definitely make things easier!

Reply:
Thanks for the information! I will just keep most of things to be the same but only change the global redirector. We can decide whether to use the specific redirector in the future.